### PR TITLE
Adds modular pc vendors

### PR DESCRIPTION
**Pull request by Wilchenx**
## About The Pull Request
Adds a new modular pc vendor that can be found generally near the science departament lobby

## Why It's Good For The Game
Right now modular pcs, even at their most basic state are locked away to only one departament that we all hate and love taking things away from, Science.

So its time for them to share more of their deluxe silicate toys.

This is also a moneysink (its almost like a """""economy""""" where money just keeps flowing in and there are no sinks for it is poorly designed)
## Changelog
:cl:
add: The modular pc vendor
del: Random plant on deltastation (his name was rodger)
del: A table with pamphlet on metastation
del: A table with wires and circuitry from pubbystation
del: Crucial emptyspace from boxstation, further cluttering the space around insulated gloves vendor
/:cl:

### CODE REVIEW

File: code/modules/vending/modularpc.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	name = "\improper Deluxe Silicon Selections"
+	desc = "All the parts you need to build your own custom pc."
+	icon_state = "modularpc"
+	icon_deny = "modularpc-deny"
+	product_ads = "Get your gamer gear!;The best GPUs for all of your space-crypto needs!;The most robust cooling!;The finest RGB in space!"
+	vend_reply = "Game on!"
+	products = list(/obj/item/modular_computer/laptop = 4,
+					/obj/item/modular_computer/tablet = 4,
+					/obj/item/computer_hardware/hard_drive = 4,
+					/obj/item/computer_hardware/hard_drive/small = 4,
```
Comment (Tlaltecuhtli) [inline code/modules/vending/modularpc.dm]: why have the downgrade at the same price and with the same availability of the better one

Comment (JStheguy) [reply code/modules/vending/modularpc.dm]: Well, regular-size hard drives won't fit into tablets, iirc.
Though it is still kind of silly, regardless. @Wilchenx if you want to change this, just set the custom_price var of either the small or normal drive to something more suitable.

Comment (Wilchenx) [reply code/modules/vending/modularpc.dm]: god i love undocumented features
image

**Comment by ArcaneMusic** [comment]
Solid hustle, maybe this will convince people to add more functionality to tablets/modular computers.

File: code/modules/vending/modularpc.dm (hunk 1 above)
Comment (optimumtact) [reply code/modules/vending/modularpc.dm]: they say as they stare at the documentation

Comment (optimumtact) [reply code/modules/vending/modularpc.dm]: can you confirm you've fixed this

Comment (Wilchenx) [reply code/modules/vending/modularpc.dm]: I've yet not changed the price, there's also still a merge conflict on donutstation

**Comment by Wilchenx** [comment]
i claim the fifth

**Pull request closed without merging**